habitat/tasks/rearrange/grip_actions.py

```diff
@@ -18,7 +19,61 @@
 )
 
 
-class GripSimulatorTaskAction(SimulatorTaskAction):
+class RobotAction(SimulatorTaskAction):
```

Comment:
Any reason the `RobotAction` is in `grip_actions.py` instead of `actions.py` ? The separation between what is in `grip_actions.py`vs `actions.py` seems kind of vague.

Reply:
`RobotAction` belongs in neither. Created a new file for this. `grip_actions` should be grip only actions, `actions.py` is everything else.